src/app/shared/forms/components/form-control-feedback/form-control-feedback.component.html

```diff
@@ -1,5 +1,7 @@
 <ng-container *ngIf="control.dirty && control.validator">
   <fa-icon class="form-control-feedback" *ngIf="this.control.valid; else invalid" [icon]="['fas', 'check']"></fa-icon>
   <ng-template #invalid> <fa-icon class="form-control-feedback" [icon]="['fas', 'times']"></fa-icon> </ng-template>
-  <small *ngFor="let e$ of errors" class="form-text"> {{ e$ | async }} </small>
+  <ng-container *ngFor="let e$ of errors; let i = index">
+    <small *ngIf="i === 0" class="form-text">{{ e$ | async }}</small>
+  </ng-container>
```

Comment:
` | slice:0:1`

Reply:
I don't quite understand why `async` is used on `e$` here. It should be the errors array that should be subscribed to.